README.md

```diff
@@ -42,12 +42,13 @@ Support for DXVK-NVAPI has been added to popular game launchers. See their respe
 
 ### Steam Play / Proton
 
-Proton 8.0 includes DXVK-NVAPI and enables it by default for a lot of titles.
+Proton 9.0 includes DXVK-NVAPI and enables it by default for all titles with a few exceptions. Proton 9.0 can force-enable DXVK-NVAPI for non-NVIDIA GPUs when setting `PROTON_FORCE_NVAPI=1`. Contrary, `PROTON_DISABLE_NVAPI` disables DXVK-NVAPI.
```

Comment:
I'd add that `PROTON_FORCE_NVAPI` can also be used to reenable dxvk-nvapi for those few games that have it disabled by default.